File: c++/lucky_straight.cpp
#include <iostream>
#include <vector>
using namespace std;

void solve() {
	string numbers;
	cin >> numbers;
	int sum = 0;
	for (int i = 0; i < numbers.size() / 2; i++)
		sum += (numbers[i] - '0');
	for (int i = numbers.size() / 2; i < numbers.size(); i++)
		sum -= (numbers[i] - '0');
	if (sum == 0)
		cout << "LUCKY" << "\n";
	else
		cout << "READY" << "\n";
	
}

int main() {
	solve();
}

File: python/lucky_straight.py
numbers = list(input())
left = numbers[0:len(numbers)//2]
right = numbers[len(numbers)//2:]
left_sum,right_sum=0,0
for i in left:
    left_sum+=int(i)
for i in right:
    right_sum+=int(i)

if left_sum==right_sum:
    print("LUCKY")
else:
    print("READY")
